source/quickstart/osx.rst

```diff
@@ -6,31 +6,213 @@ Quickstart for Mac OSX
 
 .. default-domain:: minio
 
-This page deploys MinIO on Mac OSX for use with early development
-and evaluation environments.
+.. |OS| replace:: MacOS
 
-For instructions on deploying to production environments,
-see :ref:`deploy-minio-distributed`.
+This page deploys MinIO on |OS| for use with early development and evaluation environments.
+
+For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.
+
+The procedure described here deploys MinIO in :guilabel:`Standlone Mode`` on a directory on the |OS| harddrive.
+A standalone MinIO deployment consists of a single MinIO server process with a single drive or storage volue (what we call "filesystem mode").
+The MinIO server provides an S3 access layer to the drive or volume.
+
```

Comment:
I think we are repeating ourselves a little bit here. I think we can compact 11, 15, and 16 into a single point, and then come in with 17. 

Spitballing:

This procedure deploys a single MinIO server in Standalone Mode using a directory on the local filesystem. Standalone mode ("filesystem mode") provides an S3 access layer to the drive or volume.

Standalone deployments are for use with...for instructions on production...